MANIFEST.in

```diff
@@ -3,3 +3,4 @@ include LICENSE
 include Makefile
 recursive-include avocado/etc *
 recursive-include selftests *
+recursive-include examples *
```

Comment:
I see, so you want to avoid all the `/usr/share` stuff by putting it here. Yes, that is probably more-python way (less linux way) so the first comment in the PR does not make sense. I'll think about it after finishing the first review...

Reply:
I think it does... because the configuration content `etc` is a requirement for the basic *Python* avocado module.  Just like the HTML template file is a requirement for that plugin.

Keep in mind that there's a difference between "package data" and "data" in Python lingo.

Reply:
BTW, the glob import is unnecessary now and should be dropped in this commit (fixing it on v2).